conjure-go-client/httpclient/client_test.go

```diff
@@ -32,10 +32,8 @@ import (
 
 func TestNoBaseURIs(t *testing.T) {
```

Comment:
this test shows the old behavior, where we can create a "broken" client that errors on requests